server/src/Player.ts

```diff
@@ -0,0 +1,32 @@
+import Position from "./Position";
+import Rotation from './Rotation';
+
+const distanceLimit = 5;
+const zLimit = 0.5;
+
+export default class Player {
```

Comment:
I would also add a `name` parameter, which should for now just be like `player${playerCounter++}`, for testing purposes, so that you can display that name in the client.

In the future, the client will pass this name when connecting to the server.

Reply:
Done.